### PR TITLE
[DOCFIX] Improve Fuse mount docs

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?
Improve Fuse documents about how to pass in fuse mount configuration.
### Why are the changes needed?
Tell users how to pass in fuse mount configuration.
### Does this PR introduce any user facing changes?
configuration

### CODE REVIEW

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -255,9 +258,12 @@ and the set up process may be different depending on the platform.
 
 ```console
 $ ${ALLUXIO_HOME}/integration/fuse/bin/alluxio-fuse mount \
-  -o [comma separated mount options] mount_point [alluxio_path]
+  -o [comma separated mount options] [mount_point] [alluxio_path]
 ```
 
+If the mount options is not provided through alluxio-fuse command line,
```
Comment (ssz1997) [inline docs/en/api/POSIX-API.md:264]: I think "If no mount option is provided" is more grammatically correct.

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -494,15 +499,17 @@ To change this path to `/foo/bar/alluxio-fuse` on host file system, replace `/tm
...
 {% endnavtabs %}
 
 Additional POSIX API configuration can also be added based on actual use cases.
 For example,
-- `-e "ALLUXIO_JAVA_OPTS="-Dalluxio.fuse.user.group.translation.enabled=true"` add alluxio client/fuse side configuration
+- `-e "ALLUXIO_FUSE_JAVA_OPTS="-Dalluxio.fuse.user.group.translation.enabled=true"` add alluxio client/fuse side configuration
 to Alluxio POSIX API container. The example java opts enables translating Alluxio users and groups
 into Unix users and groups when exposing Alluxio files through the FUSE API.
-- `--fuse-opts=kernel_cache,max_read=131072,attr_timeout=7200,entry_timeout=7200` add fuse mount options.
+- `-e "ALLUXIO_FUSE_JAVA_OPTS="-Dlluxio.fuse.mount.point=/mnt/alluxio-fuse-new-path"` set Alluxio Fuse mount point to a new path other than default value `/mnt/alluxio-fuse`.
```
Comment (ssz1997) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:512]: nit: to a "different" path other than "the default path"

**Review by ssz1997 — APPROVED** [review]
LGTM. Only minor suggestions.

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**